dapps/marketplace/public/template.html

```diff
@@ -38,7 +38,6 @@
     }
     if (hostname !== 'localhost' && hostname !== '0.0.0.0') {
       gtag('js', new Date());
-      gtag('config', 'UA-106384880-2');
```

Comment:
This actually records a pageview, but it's doing it incorrectly since it's not taking into account the path of the page as used by react router. Instead it's always recording "/"